docs/deployment.md

```
@@ -1,21 +1,44 @@
-## Manual
+### Building from Source (Docker)
 
-You can easily run NovaCrate without Docker on any modern operating system where Node.js 20+ is installed. Simply clone the repository and run the following commands:
+1. Clone the NovaCrate repository
+2. In [docker-compose.build.yml](/docker-compose.build.yml): Remove the line comments and set the environment variables (all of them are optional, only set those you need)
+3. Build the image: `docker compose -f docker-compose.yml -f docker-compose.build.yml build`
+4. Run using your own docker compose or by adapting the default [docker-compose.yml](/docker-compose.yml).
+
+## Deploying manually (without Docker)
+
+You can run NovaCrate without Docker on any modern operating system where Node.js 20+ is installed. Clone the NovaCrate repository and run the following commands. Note that you need to set [Environment Variables](#Environment-Variables) before the build step.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#environment-variables`; Actual: `#Environment-Variables`

(MD051, link-fragments)